api/agentpb/agent.proto

```diff
@@ -490,6 +495,7 @@ message StartJobRequest {
     // Backup target location.
     oneof location_config {
       S3LocationConfig s3_config = 10;
+      PMMClientLocationConfig pmm_client_config = 11;
```

Comment:
The proto message seems MySQL-specific, is there some reason why we added the LocationConfig here too?